<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgProductsElaborated</class>
 <widget class="QWidget" name="wdgProductsElaborated">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>654</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Elaborated products list</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Select your search</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Enter a string</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QToolButton" name="cmd">
            <property name="icon">
             <iconset resource="../images/caloriestracker.qrc">
              <normaloff>:/caloriestracker/search.png</normaloff>:/caloriestracker/search.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="myQTableWidget" name="tblProductsElaborated" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="lblFound">
       <property name="text">
        <string>Registers found</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionProductNew">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/add.png</normaloff>:/caloriestracker/add.png</iconset>
   </property>
   <property name="text">
    <string>New product</string>
   </property>
   <property name="toolTip">
    <string>New user product</string>
   </property>
  </action>
  <action name="actionProductDelete">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/list-remove.png</normaloff>:/caloriestracker/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete product</string>
   </property>
   <property name="toolTip">
    <string>Delete user product</string>
   </property>
  </action>
  <action name="actionProductEdit">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/document-edit.png</normaloff>:/caloriestracker/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit product</string>
   </property>
   <property name="toolTip">
    <string>Edit product</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txt</tabstop>
 </tabstops>
 <resources>
  <include location="../images/caloriestracker.qrc"/>
 </resources>
 <connections/>
</ui>
